DEV-Simple/Utilities/Globals.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Globals.swift
+//  DEV-Simple
+//
+//  Created by Erick Alberto Gonzales Fajardo on 1/7/19.
+//  Copyright © 2019 DEV. All rights reserved.
```

Comment:
What's up with this attribution?

Reply:
The file had the attribution from inception (Xcode's default), but it appears as brand new because I shuffled the directory structure a bit 😅 I'm okay with leaving it as is